.gitignore

```diff
@@ -20,6 +20,7 @@ vendor/
 # VS Code
 .history/
 .vscode/
+__debug_bin
```

Comment:
😄

Reply:
Not sure what happened...but vs code isn't cleaning up after debug sessions anymore.